core/templates/dev/head/profile/Preferences.js

```diff
@@ -35,6 +36,46 @@ oppia.controller('Preferences', ['$scope', '$http', '$rootScope', '$modal', '$ti
     _saveDataItem('user_bio', userBio);
   };
 
+  $scope.changedAtLeastOnce = false;
+  $scope.subjectInterestsWarningText = null;
+  $scope.TAG_REGEX = '^[a-z ]+$';
+
+  $scope.updateSubjectInterestsWarning = function(subjectInterests) {
+    var alpha = /^[A-Za-z]+$/;
```

Comment:
Are you referring to: $scope.newChoiceValidator = new RegExp('^[a-z ]+$');

Also, can't I just input a regex like this in preferences.html (this is the format in exploration_settings.html):
new-choice-regex="^[a-z ]+$" width="100%"